README.md

```diff
@@ -42,31 +42,73 @@ jobs:
 
-The default configuration makes it possible to skip lint, formatting, or breaking change checks on a PR
-by adding a label with a (case-insensitive) special name to that PR.
+### Configuration
 
-- `buf skip breaking`: skips breaking change detection.
-- `buf skip lint`: skips lint.
-- `buf skip format`: skips format. 
+To customize the behavior of the action, you can set the following inputs in the workflow file.
+Add these inputs under the `with` section of the `uses` step in the workflow file.
+
+```yaml
+- uses: bufbuild/buf-action@v0.1
+  with:
+    ...
+```
+
+| Parameter                       | Description                                        | Default            |
+|:--------------------------------|:---------------------------------------------------|:-------------------|
+| `archive`                       | Whether to run the archive step. | Runs on Git deletes |
+| `breaking`                      | Whether to run the breaking change detection step. | Runs on pushes to Git PR |
+| `breaking_against`              | [Input](https://buf.build/docs/reference/inputs) to compare against. | Base of the PR or the commit before the event |
+| `domain`                        | Domain for logging into the BSR, enterprise only.| `buf.build` |
+| `exclude_imports`               | Exclude imports. | |
+| `exclude_paths`                 | Exclude specific files or directories (separated by newlines). | |
+| `format`                        | Whether to run the formatting step. | Runs on pushes to Git PR |
+| `input`                         | [Input](https://buf.build/docs/reference/inputs) for the `buf` command. | |
+| `lint`                          | Whether to run the linting step. | Runs on pushes to Git PR |
+| `paths`                         | Limit to specific files or directories (separated by newlines). | |
+| `pr_comment`                    | Comment the results on the pull request. | Only on pull requests |
+| `push`                          | Whether to run the push step. | Runs on Git pushes |
+| `push_disable_create`           | Disables repository creation if it does not exist. | False |
+| `setup_only`                    | Setup only the `buf` environment, optionally logging into the BSR, but without executing other commands. | |
+| `token`                         | API token for [logging into the BSR](https://buf.build/docs/bsr/authentication). | |
+| `username`                      | Username for [logging into the BSR](https://buf.build/docs/bsr/authentication). | |
+| `version`                       | Version of the `buf` CLI to use. | Latest [version][buf-releases] |
+
+
+### Skip the breaking change detection step
+
+By default, the action runs the breaking change detection step on every pull request.
+To skip this step, add the label `buf skip breaking` to the PR.
+
+![Skip breaking changes example](./static/img/skip-breaking-example.png "Skip breaking changes example")
```

Comment:
This screenshot doesn't really demonstrate the breaking check being skipped to me. I know we don't get a nice GitHub action skipped status we can show since we are running these all in one check, but wondering if we can show a screenshot that leverages our summary comment to show this. here is my proposal:

<img width="500" alt="Screenshot 2024-06-28 at 4 29 03 PM" src="https://github.com/bufbuild/buf-action/assets/754768/f3168a68-ae49-4d0f-b63e-45733a4cb082">

Reply:
The proportions on the image will render very weird as its always the full width. Need to prefer landscape over portrait images.
<img width="838" alt="Screenshot 2024-06-28 at 4 53 11 PM" src="https://github.com/bufbuild/buf-action/assets/3036610/1f54923b-e81f-4319-871a-56106b191a82">

Reply:
What about updating the PR comment to show the breaking skipped. And maybe include the previous commit with a green check?

Reply:
I'll have a play with it.

Reply:
Updated with an example. Annoyingly we don't show the breaking skipped anymore, it gets dropped from the status. Will have a think how to fix that as would be nice to show the step did run before.